src/components/UI/inputs/AppInput/AppInput.tsx

```diff
@@ -1,5 +1,5 @@
-import React, { ChangeEventHandler, useEffect, useState } from 'react'
 import { Input } from 'antd'
+import React, { useEffect, useState } from 'react'
```

Comment:
:)

Reply:
Should I have refactored it or something?